docs/src/contributing/testnet.md

```diff
@@ -0,0 +1,237 @@
+# Test Network Management
```

Comment:
Are we sure we want to migrate this out of `net/README.md` ? Maybe a pointer from the relevant place in docs --> the README would make sense

Reply:
I was actually thinking of keeping READMEs like this one as symlinks to the docs markdown files. This way they are always in sync, and all relevant information is nicely presented in the web-based docs which are nicer to look at if you are e.g. using console-based editor for coding.  But it could be a separate PR as well.

Reply:
yeah, that makes sense to me. Iterative PRs are my preference

Reply:
Similar to feedback on this README, I'm thinking how we can ensure the docs/contributing stuff stays consistent and doesn't duplicate items in the https://github.com/anza-xyz/agave/blob/master/CONTRIBUTING.md file

Reply:
Uhhh I have missed that one entirely.  Should I merge it with the pull-request.md and link it same as the testnet.md?

Reply:
I think that makes sense, but curious what @nickfrosty and @jacobcreech think